sirepo/package_data/static/js/sirepo-plotting.js

```diff
@@ -3053,12 +3098,13 @@ SIREPO.app.directive('heatmap', function(appState, layoutService, plotting, util
             };
         },
         link: function link(scope, element) {
+            plotting.addWatermark(authState);
```

Comment:
plotting can ask for authState on its own, can't it? No need to cascade here.

Reply:
4e62436